### ISSUE TITLE
[Support]: 

### ISSUE THREAD

**Issue by android52** [blakeblackshear/frigate#2097]
### Describe the problem you are having

I upgrade from v 0.9.1-800f33e to v 0.9.2-25bb515 and one of my cameras started displaying as a green screen in Frigate.
I have also tried v 0.9.3-57645c6-amd64 and get the same green screen.

I presume from the logs that I am missing tags or have extra arguments, or there are settings that are now needed that weren't before, but am not sure what is needed or how to configure it.  The camera output is h.265 so perhaps I think from this "Video codec hevc not compatible with flv" in the logs I need to convert the stream, but did this change in 9.2?

This is running on my Synology NAS in docker, and when rolling back to v0.9.1 it is fine.

The logs from 9.2 (and 9.3 is similar) are showing as below (two iterations are shown, this repeats until the container is stopped):

Any suggestions on what I'm missing

### Version

0.9.2-25bb515

### Frigate config file

```yaml
mqtt:
  host: 192.168.2.5
  port: 1883
  topic_prefix: frigate
  client_id: frigate
  user: mqttha
  password: <mqtt password>
  stats_interval: 60

cameras:
  frontdoorfrig:
    ffmpeg:
      inputs:
        - path: <path to front door camera>
          roles:
            - detect

    detect:
      width: 1536
      height: 2048
      fps: 5

    motion:
      mask:
        - 0,0,0,417,0,1265,278,1256,294,1025,672,1030,781,908,1016,910,957,1049,948,1348,846,1635,1536,1699,1536,448,1536,0
    objects:
      track:
        - person
    snapshots:
      enabled: True
      timestamp: False
      bounding_box: False
      crop: False
      required_zones: []
      retain:
        default: 11
        objects:
          person: 1
    zones:
      Front_Porch_Zone:
        coordinates: 662,1277,576,1332,249,1223,0,1288,58,1473,0,1497,0,2048,1536,2048,1536,1617,851,1635,946,1344,956,1158,714,1144
    rtmp:
      enabled: True

  kitchenwindowfrig:
    ffmpeg:
      inputs:
        - path: <path to kitchen window camera>
          roles:
            - detect

    detect:
      width: 2304
      height: 1296
      fps: 5

    motion:
      mask:
        - 1377,207,2304,258,2304,0,0,0,0,290,207,262,954,233
        - 1304,713,1400,712,1385,653,1532,534,1492,462,1559,422,1689,381,1785,437,1853,364,1770,582,1788,690,1758,776,1789,990,1852,1094,1815,1296,0,1296,0,768,136,616,361,608,392,708,502,638,529,609,595,682,635,661,857,718,826,780,962,783,1014,671,1300,621
        - 2304,422,2304,802,2304,952,2206,890,2059,1006,1970,745,1983,552,1984,422,2103,298
    objects:
      track:
        - person
        - car
      filters:
        car:
          mask:
            - 0,0,2304,0,2304,414,1521,391,946,384,939,233,439,267,442,311,96,447,0,455
            - 0,1296,2304,1296,2304,716,108,729,98,568,0,566
        person:
          mask:
            - 1377,207,2304,258,2304,0,0,0,0,214,201,211,205,281,954,233
    snapshots:
      enabled: True
      timestamp: False
      bounding_box: False
      crop: False
      required_zones: []
      retain:
        default: 11
        objects:
          person: 1
          car: 1
    zones:
      Front_Porch_Zone:
        coordinates: 1642,900,0,902,0,1296,2304,1296,2304,1210,2013,923,1748,932

detectors:
  cpu1:
    type: cpu
  cpu2:
    type: cpu
```

### Relevant log output

```shell
,[services.d] starting services
,[services.d] done.
,[2021-10-24 01:13:56] frigate.app                    INFO    : Starting Frigate (0.9.2-25bb515)
,[2021-10-24 01:13:56] frigate.app                    INFO    : Creating directory: /tmp/cache
,Starting migrations
,[2021-10-24 01:13:56] peewee_migrate                 INFO    : Starting migrations
,There is nothing to migrate
,[2021-10-24 01:13:56] peewee_migrate                 INFO    : There is nothing to migrate
,[2021-10-24 01:13:56] frigate.mqtt                   INFO    : MQTT connected
,[2021-10-24 01:13:56] frigate.app                    INFO    : Output process started: 218
,[2021-10-24 01:13:56] ws4py                          INFO    : Using epoll
,[2021-10-24 01:13:56] detector.cpu1                  INFO    : Starting detection process: 216
,[2021-10-24 01:13:56] frigate.app                    INFO    : Camera processor started for frontdoorfrig: 225
,[2021-10-24 01:13:56] frigate.app                    INFO    : Camera processor started for kitchenwindowfrig: 226
,[2021-10-24 01:13:56] frigate.app                    INFO    : Capture process started for frontdoorfrig: 227
,[2021-10-24 01:13:56] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
,[2021-10-24 01:13:56] detector.cpu2                  INFO    : Starting detection process: 217
,[2021-10-24 01:13:56] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
,[2021-10-24 01:13:56] frigate.app                    INFO    : Capture process started for kitchenwindowfrig: 228
,[2021-10-24 01:13:56] ws4py                          INFO    : Using epoll
,[2021-10-24 01:14:04] frigate.video                  INFO    : kitchenwindowfrig: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
,[2021-10-24 01:14:04] frigate.video                  INFO    : kitchenwindowfrig: ffmpeg process is not running. exiting capture thread...
,[2021-10-24 01:14:09] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:48622]
,[2021-10-24 01:14:17] watchdog.kitchenwindowfrig     ERROR   : FFMPEG process crashed unexpectedly for kitchenwindowfrig.
,[2021-10-24 01:14:17] watchdog.kitchenwindowfrig     ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
,[2021-10-24 01:14:17] watchdog.kitchenwindowfrig     ERROR   : You may have invalid args defined for this camera.
,[2021-10-24 01:14:17] ffmpeg.kitchenwindowfrig.detect ERROR   : [flv @ 0x55c21aaab180] Video codec hevc not compatible with flv
,[2021-10-24 01:14:17] ffmpeg.kitchenwindowfrig.detect ERROR   : Could not write header for output file #0 (incorrect codec parameters ?): Function not implemented
,[2021-10-24 01:14:17] ffmpeg.kitchenwindowfrig.detect ERROR   : 
,[2021-10-24 01:14:20] frigate.video                  INFO    : kitchenwindowfrig: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
,[2021-10-24 01:14:20] frigate.video                  INFO    : kitchenwindowfrig: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
,[2021-10-24 01:14:20] frigate.video                  INFO    : kitchenwindowfrig: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
,[2021-10-24 01:14:20] frigate.video                  INFO    : kitchenwindowfrig: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
,[2021-10-24 01:14:20] frigate.video                  INFO    : kitchenwindowfrig: ffmpeg process is not running. exiting capture thread...
,[2021-10-24 01:14:27] watchdog.kitchenwindowfrig     ERROR   : FFMPEG process crashed unexpectedly for kitchenwindowfrig.
,[2021-10-24 01:14:27] watchdog.kitchenwindowfrig     ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
,[2021-10-24 01:14:27] watchdog.kitchenwindowfrig     ERROR   : You may have invalid args defined for this camera.
,[2021-10-24 01:14:27] ffmpeg.kitchenwindowfrig.detect ERROR   : [flv @ 0x556ddd0cb180] Video codec hevc not compatible with flv
,[2021-10-24 01:14:27] ffmpeg.kitchenwindowfrig.detect ERROR   : Could not write header for output file #0 (incorrect codec parameters ?): Function not implemented
```

### FFprobe output from your camera

```shell
Sorry - I'm not sure how to do this.

I tried from the portainer console window "ffprobe http://192.168.2.5:5005/cameras/kitchenwindowfrig" and the return "Connection refused" (note port 5005 maps to port 5000 in docker)
```

### Frigate stats

```json
These are from rolled back 9.1:

{"detection_fps":0.0,"detectors":{"cpu1":{"detection_start":0.0,"inference_speed":134.7,"pid":216},"cpu2":{"detection_start":0.0,"inference_speed":137.78,"pid":217}},"frontdoorfrig":{"camera_fps":5.0,"capture_pid":227,"detection_fps":0.0,"pid":224,"process_fps":5.0,"skipped_fps":0.0},"kitchenwindowfrig":{"camera_fps":5.0,"capture_pid":228,"detection_fps":0.0,"pid":226,"process_fps":5.0,"skipped_fps":0.0},"service":{"storage":{"/dev/shm":{"free":56.9,"mount_type":"tmpfs","total":67.1,"used":10.2},"/media/frigate/clips":{"free":8403648.8,"mount_type":"btrfs","total":13428362.9,"used":5024714.1},"/media/frigate/recordings":{"free":8403648.8,"mount_type":"btrfs","total":13428362.9,"used":5024714.1},"/tmp/cache":{"free":8403648.8,"mount_type":"btrfs","total":13428362.9,"used":5024714.1}},"uptime":936,"version":"0.9.1-800f33e"}}
```

### Operating system

Other Linux

### Install method

Docker CLI

### Coral version

CPU (no coral)

### Network connection

Wired

### Camera make and model

Heimvision HM202A (Kitchen Window), EzViz DB1 (Front Door)

**Comment by blakeblackshear** [comment]
Add this under the `kitchenwindowfrig` camera:
```yaml
rtmp:
  enabled: False
```

Just pushed up the 0.9.3 release builds, so give that one a try.

**Comment by android52** [comment]
Thank you.  Thank you.  Thank you.  

Was not expecting a response on a Saturday night - so thank you.

That seems to have fixed it - running 0.9.3-5e692ac and the Kitchen Window is showing the live image, not the green screen.

Thank you for this, and for Frigate - really love it.
